jobs/config.json

```diff
@@ -9907,6 +9907,27 @@
       "UNKNOWN"
     ]
   },
+  "mlkube-build-presubmit": {
+    "args": [
+      "--charts",
+      "--check-leaked-resources",
+      "--deployment=gke",
+      "--extract=gke",
+      "--gcp-cloud-sdk=gs://cloud-sdk-testing/ci/staging",
+      "--gcp-node-image=gci",
+      "--gcp-project=mlkube-testing",
```

Comment:
for the presubmit error you need to add something to https://github.com/kubernetes/test-infra/blob/master/jobs/config_test.py#L681-L791